_includes/v20.1/app/hibernate-basic-sample/build.gradle

```diff
@@ -12,5 +12,5 @@ repositories {
 
 dependencies {
     implementation 'org.hibernate:hibernate-core:5.4.19.Final'
-    implementation 'org.postgresql:postgresql:42.2.14'
+    implementation 'org.postgresql:postgresql:42.2.9'
```

Comment:
FWIW, the latest version is 42.2.18, and that's what I used in the Java hello world examples.

Reply:
Right on. I think we just want to make sure we're in sync with whichever version the official test suite is running, as that's going to be quite a bit more rigorous than our example apps. I ran this Liquibase project on 4.2.18 as well without issues. And I think we've used 42.2.14 elsewhere with no issues.

@rafiss Perhaps we should look into updating the JDBC driver that we are using on our tests? (unless there's a specific reason why we are using 42.2.9)